config/core_pools_whitelist.json

```diff
@@ -1,10 +1,8 @@
 {
   "mainnet": {
     "0xxxx": "WHITELIST-ME",
-    "0x1e19cf2d73a72ef1332c882f20534b6519be0276000200000000000000000112": "B-rETH-STABLE",
     "0xf7a826d47c8e02835d94fb0aa40f0cc9505cb1340002000000000000000005e0": "ECLP-wstETH-cbETH",
     "0xf16aee6a71af1a9bc8f56975a4c2705ca7a782bc0002000000000000000004bb": "20WETH-80ALCX",
-    "0xb08885e6026bab4333a80024ec25a1a3e1ff2b8a000200000000000000000445": "B-staFiETH-WETH-Stable",
     "0x8353157092ed8be69a9df8f95af097bbf33cb2af0000000000000000000005d9": "GHO/USDT/USDC",
     "0x5f1f4e50ba51d723f12385a8a9606afc3a0555f5000200000000000000000465": "50wstETH-50LDO",
     "0x1ee442b5326009bb18f2f472d3e0061513d1a0ff000200000000000000000464": "50rETH-50BADGER",
```

Comment:
I don't think rETH-BADHER nor wstETH-LDO need a special exception from the core pools list. They're both not getting anything anyway due to not enough fees:
https://docs.google.com/spreadsheets/d/16axBQBVNdPssQyjcIyb_4m4vOb5rKISIlrVMk7A2qjs/edit#gid=748841216

Reply:
i would suggest to move the whitelist pruning/growing discussion to a separate pr

im trying to come up with the logic of determining what a core pool is, and just want to make sure that the resulting json isnt missing any entries compared to the existing manual json.

(dont get me wrong, i agree, the pools you mention obviously have too low tvl)